eurc/solana/metadata.json

```diff
@@ -2,5 +2,5 @@
   "name": "EURC",
   "symbol": "EURC",
   "description": "EURC is a fully collateralized euro stablecoin.",
-  "image": "https://www.circle.com/hubfs/Brand/EURC/EURC-icon_128x128.png"
+  "image": "https://raw.githubusercontent.com/centrehq/centre-ecosystem/master/logos/eurc.png"
```

Comment:
do we need this for usdc?

Reply:
no, because USDC is using the deprecated version of metadata handling where the metadata lives in https://github.com/solana-labs/token-list. I believe the usdc metadata in that repo points a USDC icon hosted under a centre link, so it is unaffected by the marketing website migration. This repo became read-only when we launched eurc, so the metadata for eurc lives on-chain, and it points to this repo. 

see here: https://explorer.solana.com/address/HzwqbKZw8HxMN6bF2yFZNrht3c2iXXzpKcFu7uBEDKtr/metadata
then click Metadata.
![Screenshot 2024-10-25 at 2 48 06 PM](https://github.com/user-attachments/assets/e0adbb6d-64f3-40db-860f-f8c28aa02989)